hwcomposer/wayland-hwc.cpp

```diff
@@ -710,34 +710,49 @@ pointer_handle_axis(void *data, struct wl_pointer *,
     struct display* display = (struct display*)data;
     struct input_event event[2];
     struct timespec rt;
-    unsigned int res, n = 0;
+    unsigned int res, v, n = 0;
+    static const int step = 4000;
```

Comment:
Kinda arbitrary, but i think it should be a decent default. Please test on your touchpads!